app/src/organisms/CommandText/PipettingCommandText.tsx

```diff
@@ -87,37 +85,36 @@ export const PipettingCommandText = ({
       const { flowRate } = command.params
       return t('blowout', {
         well_name: wellName,
-        labware: getLabwareName(robotSideAnalysis, labwareId),
+        labware: getLabwareName(commandTextData, labwareId),
         labware_location: displayLocation,
         flow_rate: flowRate,
       })
     }
     case 'dropTip': {
-      const loadedLabware = getLoadedLabware(robotSideAnalysis, labwareId)
+      const loadedLabware = getLoadedLabware(commandTextData, labwareId)
       const labwareDefinitions = getLabwareDefinitionsFromCommands(
-        robotSideAnalysis.commands
+        commandTextData.commands
       )
       const labwareDef = labwareDefinitions.find(
         lw => getLabwareDefURI(lw) === loadedLabware?.definitionUri
       )
       return labwareDef?.parameters.isTiprack
         ? t('return_tip', {
             well_name: wellName,
-            labware: getLabwareName(robotSideAnalysis, labwareId),
+            labware: getLabwareName(commandTextData, labwareId),
             labware_location: displayLocation,
           })
         : t('drop_tip', {
             well_name: wellName,
-            labware: getLabwareName(robotSideAnalysis, labwareId),
+            labware: getLabwareName(commandTextData, labwareId),
           })
     }
     case 'pickUpTip': {
       const pipetteId = command.params.pipetteId
       const pipetteName:
         | PipetteName
-        | undefined = robotSideAnalysis.pipettes.find(
-        pip => pip.id === pipetteId
-      )?.pipetteName
+        | undefined = commandTextData.pipettes.find(pip => pip.id === pipetteId)
```

Comment:
nit
can we use `pipette`? 🤣 

```suggestion
        | undefined = commandTextData.pipettes.find(pipette => pipette.id === pipetteId)
```

Reply:
for sure